src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
+
+/**
+ * Resolve a `description` key from config.
+ * Process it through Markdown.
+ */
+module.exports = function description(ctx) {
+  var desc = ctx.description;
+
+  if (isPath(desc)) {
+    var contents = tryLoadFile(desc, ctx);
+    ctx.description = contents ? marked(contents) : contents;
+    return;
+  }
+
+  if (isString(desc)) {
```

Comment:
I don't think there is much point in adding a `isString` function just for this.